src/pkcs15init/pkcs15-lib.c

```diff
@@ -1238,7 +1238,13 @@ sc_pkcs15init_init_prkdf(struct sc_pkcs15_card *p15card, struct sc_profile *prof
 		key_info->params.data = &keyargs->key.u.ec.params;
 		key_info->params.free_params = sc_pkcs15init_empty_callback;
 		key_info->field_length = ecparams->field_length;
-		key_info->modulus_length = 0;
```

Comment:
Right, no need do modify code here, we will do it in pkcs15-epass2003.c file, will submit another pull request